### ISSUE TITLE
12 vs 24 format in datetime-picker

### ISSUE THREAD

**Issue by chisholmd** [fooloomanzoo/datetime-picker#13]
Ugh....was having trouble getting 24 hour to work until I noticed that I was using "native" and the implementation of input "type=datetime-local" is fubar.  :(

I have one request though.  When I drop down the calendar and I select a date and time I was confused about how to confirm my selection.   I have a graphic to explain.
![cal-ok](https://user-images.githubusercontent.com/13528823/31319103-537777a2-ac12-11e7-9574-fffbfd11d6de.jpg)

**Comment by fooloomanzoo** [comment]
@chisholmd 

About 24-hour-format, for native inputs I can not find anything about on how to influence or force that behavior. In most local formats, the 24-hour format is standard, so I made hour-12 optional for the polyfill.

About the buttons, I see what you mean and I changed the button to a check mark and moved the buttons section inside the pickers.

![image](https://user-images.githubusercontent.com/9318952/31343177-2546b4a8-ad0f-11e7-9abb-d549ef385185.png)

Regards!
